{{cookiecutter.projectDirectory}}/generateModels

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd $(dirname $0)
+
+./APIGeneration/make
```

Comment:
Why is there an extra script for the generation? Am I missing something or could this be simplified to one script?

Reply:
I thought it makes sense to have everything generation related in an own folder and this script is just for convenience from the root folder - we can also move everything in here if you want